content/en/getting_started/synthetics/api_test.md

```diff
@@ -105,33 +105,83 @@ When you're ready to run your test, click **Save Test**.
 
 ### Configure the test
 
-When you create a [Multistep API test][12], you can define a sequence of HTTP requests and create variables from the response data to re-inject their values in subsequent steps. 
+When you create a [multistep API test][12], you can define a sequence of HTTP requests and create variables from the response data to re-inject their values in subsequent steps. 
 
-1. In the Datadog application, hover over **[UX Monitoring][3]** in the left hand menu and select **[Synthetic Tests][3]**.
-2. In the top right corner, click the **New Test** button.
-3. Select **[Multistep API test][5]**.
-4. Name, tag and select locations for your [Multistep API test][5] like you would with a single [API test][1].
-5. In **Define steps**, click **Create Your First Step**. 
+1. In **New Test** > **[Multistep API test][5]**, add a name, select a tag, and select locations for your [Multistep API test][5].
+2. In **Define steps**, click **Create Your First Step**. 
 
-    - Add a name to your step.
-    - Specify the URL you want to query. 
+    - Add a name to your step such as `Get a cart`.
+    - Specify the URL you want to query. If you don’t know what to start with, select `POST` and use `https://api.shopist.io/carts`, a test application.
     - Select **Advanced Options** to add custom request headers, authentication credentials, body content, or cookies.
     - Click **Test URL**. 
-    - Optionally, add new assertions, define execution parameters, and extract variables from the response content. 
-    - Enter a name for your variable and click on a parsed value in the **Response Body** to set the variable value. 
-    - Click **Save Step**.
-
-6. Click **Add Another Step** to add another step. By default, you can create up to ten steps.
-7. Optionally, type `{{` in the **URL** and a list of extracted and global variables appears. 
+   
+   Optionally, add an assertion based on information in the **Response Body** or click inside the **Response Header**.
```

Comment:
These options look out of place without being added to the indented list. Let's add to the existing list above.